python-server.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
yes this is to avoid implicitly running python3 and having the server fail at the print statement :) another good fix would be to make it python3 compatible 😁